<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>360</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">GS100</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="browseLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Путь до папки установленного аддона</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="browseLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Выберите сервер</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="threadOfFateCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string notr="true">Нить Судьбы</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="starOfFortuneCheckBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string notr="true">Звезда Удачи</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="eternalCallCheckBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string notr="true">Вечный Зов</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="godsLegacyCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string notr="true">Наследие Богов</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="youngGuardCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string notr="true">Молодая Гвардия</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>309</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить рейтинги</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
